### PR TITLE
Navigation Drawer Updates

### PR DESCRIPTION
**Pull request by nucleogenesis**
### Summary

Updating the navigation drawer design per [this Notion project](https://www.notion.so/learningequality/Navigation-drawer-updates-af739a97171c4ba683414c5726085688)

[Detailed Figma Designs by Khang](https://www.figma.com/file/GRjPvmnIfylaJW5iu4ToYy/0.13-Appbar-updates?node-id=0%3A1)

[Brief Video of the Changes](https://drive.google.com/file/d/18GPBZIKBeqV8NsQCGhjjcZ9YcvrZP6Pu/view?usp=sharing)

### Reviewer guidance

- Compare the styles to those noted in the Notion page above.

### References

https://www.notion.so/learningequality/Navigation-drawer-updates-af739a97171c4ba683414c5726085688

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/core/assets/src/views/CoreMenu/CoreMenuOption.vue [hunk 1]
```diff
@@ -121,68 +139,33 @@
...
+    height: 48px;
+    padding-top: 4px;
+    padding-bottom: 4px;
+    font-size: 16px;
+    text-decoration: none;
   }
-
-  .ui-menu-option-content {
+  .core-menu-option-content {
+    display: -webkit-box;
```
Comment (jonboiser) [inline kolibri/core/assets/src/views/CoreMenu/CoreMenuOption.vue]: you can remove these vendor prefixes. i think they should be added by autoprefixer.

**Comment by jonboiser** [comment]
Visually, the updates look good. Also noted that since #5884 (no keyboard focus indicator) hasn't been resolved, that I don't get any feedback when focusing on the links with my keyboard.

But I don't think it's a rule that we should treat keyboard focus as the same as mouse-hover focus, since that's not how the tab links or buttons work either

**Comment by jonboiser** [comment]
Fixed: ~The text in the bar looks lighter than black to me. In dev tools, it's `rgb(117, 117, 117)`~

The mockups say for unselected items, the text should just be 'black' (there's another gray variant color there that is crossed off now).

Screen Shot 2019-09-12 at 3 16 17 PM

**Review by khangmach — APPROVED** [review]

**Comment by indirectlylit** [comment]
w00t!

**Pull request merged**